client/src/main/java/org/apache/falcon/cli/FalconCLI.java

```diff
@@ -48,8 +48,10 @@
 
     public static final String ENV_FALCON_DEBUG = "FALCON_DEBUG";
     public static final String DEBUG_OPTION = "debug";
+    public static final String DEBUG_OPTION_DESCRIPTION = "Use debug mode to see debugging statements on stdout";
```

Comment:
Ignore my previous comment. It is better to keep the option and its description together.